nibabel/nifti1.py

```diff
@@ -519,7 +519,7 @@ def get_sizeondisk(self):
         return np.sum([e.get_sizeondisk() for e in self])
 
     def __repr__(self):
-        s = "Nifti1Extensions(%s)" % ', '.join(str(e) for e in self)
+        s = f"Nifti1Extensions({', '.join(str(e) for e in self)})"
         return s
```

Comment:
This is harder to read. I would suggest we revert. Not sure why this was a two-step method, though...

```suggestion
        return "Nifti1Extensions(%s)" % ', '.join(str(e) for e in self)
```